README.md

```diff
@@ -141,6 +142,24 @@ exclude:
 >rake preview:all
 >```
 
+## Docker (docker-compose)
+
+Docker provide an easy and fastr way to build the documentation for developers. There is no need to install any
+ruby dependency manually. The provided setup starts Jekyll inside the container in watch mode.
+Every change of a markdown file will automatically trigger a rebuild of the documentation.
+
+1. Install docker
+
+1. Run docker-compose command:
+
+```bash
+docker-compose up
+```
+
+Open your browser (http://127.0.0.1:4000)
```

Comment:
```suggestion
1. Use the server address URL provided by the system response.

   For example:
   
   ```terminal
   jekyll_1  |     Server address: http://127.0.0.1:4000/
   jekyll_1  |   Server running... press ctrl-c to stop.
   ```
```